src/clr-addons/daterangepicker/directives/daterangepicker-container/daterangepicker-container.component.ts

```diff
@@ -83,11 +79,23 @@ export class ClrDaterangepickerContainerComponent
   @Input()
   public presetsDateTime: Array<DateTimerangePreset> = [];
 
+  _timeSelection = false;
   @Input()
-  public timeSelection = false;
+  public set timeSelection(value: boolean) {
+    if (value) {
```

Comment:
false is also not true. i would just transform the value to a boolean and always set it
= !!value